02-oop-inheritance/Module_pattern/script.js

```diff
@@ -0,0 +1,57 @@
+
+var inheritPrototype = function (childClass, parentClass){
+  var copyOfParent = Object.create(parentClass.prototype);
+  copyOfParent.constructor = childClass;
+  childClass.prototype = copyOfParent;
+};
+
```

Comment:
Take a look at [this](http://addyosmani.com/resources/essentialjsdesignpatterns/book/#mixinpatternjavascript) to unveil the marvelous mysteries dancing around mixins. ;)